src/structures/Webhook.js

```diff
@@ -115,6 +115,7 @@ class Webhook {
       for (const i in options.files) {
         let file = options.files[i];
         if (typeof file === 'string') file = { attachment: file };
```

Comment:
Wouldn't it be better to combine the logic instead since you are setting file to the same thing?

`if (typeof file === 'string' || Buffer.isBuffer(file)) file = { attachment: file };`